modules/landing-pages/module.php

```diff
@@ -296,7 +205,78 @@ private function admin_localize_settings( $settings ) {
 		return array_replace_recursive( $settings, $additional_settings );
 	}
 
+	private function register_landing_page_cpt() {
+		$labels = [
+			'name' => __( 'Landing Pages', 'elementor' ),
+			'singular_name' => __( 'Landing Page', 'elementor' ),
+			'add_new' => __( 'Add New', 'elementor' ),
+			'add_new_item' => __( 'Add New Landing Page', 'elementor' ),
+			'edit_item' => __( 'Edit Landing Page', 'elementor' ),
+			'new_item' => __( 'New Landing Page', 'elementor' ),
+			'all_items' => __( 'All Landing Pages', 'elementor' ),
+			'view_item' => __( 'View Landing Page', 'elementor' ),
+			'search_items' => __( 'Search Landing Pages', 'elementor' ),
+			'not_found' => __( 'No landing pages found', 'elementor' ),
+			'not_found_in_trash' => __( 'No landing pages found in trash', 'elementor' ),
+			'parent_item_colon' => '',
+			'menu_name' => __( 'Landing Pages', 'elementor' ),
+		];
+
+		$args = [
+			'labels' => $labels,
+			'public' => true,
+			'show_in_menu' => 'edit.php?post_type=elementor_library&tabs_group=library',
+			'capability_type' => 'page',
+			'taxonomies' => [ Source_Local::TAXONOMY_TYPE_SLUG ],
+			'supports' => [ 'title', 'editor', 'comments', 'revisions', 'trackbacks', 'author', 'excerpt', 'page-attributes', 'thumbnail', 'custom-fields', 'post-formats', 'elementor' ],
+		];
+
+		if ( '/%postname%/' === $this->permalink_structure ) {
```

Comment:
It should work also for `'pages/%postname%/'`, so check with `strpos`.

Reply:
why?

Reply:
Because it should work like a page. if the page URL is `pages/page-a` the landing page URL should be `pages/landing-a`

Reply:
Done.